signalfx/resource_signalfx_slo_chart.go

```diff
@@ -0,0 +1,122 @@
+// Copyright Splunk, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package signalfx
+
+import (
+	"context"
+	"encoding/json"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	chart "github.com/signalfx/signalfx-go/chart"
+)
+
+func sloChartResource() *schema.Resource {
+	return &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"slo_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "ID of the attached SLO",
+			},
+			"url": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "URL of the chart",
+			},
+		},
+
+		Create: slochartCreate,
+		Read:   slochartRead,
+		Update: slochartUpdate,
+		Delete: slochartDelete,
```

Comment:
Please us the `<Operation>Context` methods since these methods are deprecated by the API.